airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/source.py

```diff
@@ -336,6 +336,63 @@ class AdCreativeAnalytics(LinkedInAdsAnalyticsStream):
     pivot_by = "CREATIVE"
 
 
+class AdCreativeName(Creatives):
+
+    endpoint = "adCreativesV2"
+    search_param = "projection"
+    search_param_value = "(variables(data(*,com.linkedin.ads.SponsoredUpdateCreativeVariables(*,share~(subject,text(text),content(contentEntities(*(description,entityLocation,title))))))))"
+    parent_values_map = {"creative_id": "id"}
+    parent_stream = Creatives
+
+    sync_mode = SyncMode.full_refresh
+
+    @property
+    def source_defined_cursor(self) -> bool:
+        return False
+
+    @property
+    def cursor_field(self) -> str:
+        return ""
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        return {}
+
+    def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]) -> Mapping[str, Any]:
+        return {}
+
+    def request_params(
+        self, stream_state: Mapping[str, Any], next_page_token: Mapping[str, Any] = None, **kwargs
+    ) -> MutableMapping[str, Any]:
+        return {self.search_param: self.search_param_value}
```

Comment:
do we actually need these methods after we inherit them from the Creatives class?

Reply:
Methods like `source_defined_cursor` and `cursor_field` can only be removed. Other methods
1. `next_page_token`
2. `get_updated_state`
3. `request_params`

Needs to to added, as their usage in `Creative` class is different from what we require for this stream.